### ISSUE TITLE
Upgrade PclZip [Request]

### ISSUE THREAD

**Issue by horus68** [salesagility/SuiteCRM#3101]
#### Issue

PclZip version 2.4 (November 2004) in \include\pclzip 

#### Expected Behavior

Latest version for PclZip 2.8.2 (August 2009)
See: http://www.phpconcept.net/pclzip
8 years without an update?!

#### Actual Behavior

PclZip version 2.4 is shipped with SuiteCRM 7.8.1

**Comment by gunnicom** [comment]
In include/utils/zip_utils.php there is a line:
"Use of PCLZip has been deprecated. Please enable the zip extension in your PHP install "

So maybe its best to remove it completely by now.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Upgrade PclZip [Request]](https://trello.com/c/gMQ48QpX/29-upgrade-pclzip-request)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
